src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -527,7 +528,10 @@ std::string postUpdateContext
     ciP->httpStatusCode = httpStatusCode;
   }
 
-  foundAndNotFoundAttributeSeparation(upcrsP, upcrP, ciP);
+  if (ciP->apiVersion == "v1") // FIXME PR: v2 has no forwaring yet - I imagine this function is not necessary for v2 ...
```

Comment:
I tend to use P11 for the same :)

Thus, I understand that this will be fixed in this PR (and a "Fixed in XXX" comment added to this thread).